bin/pull-prod-admin-config

```diff
@@ -0,0 +1,147 @@
+#! /bin/bash
+
+# DOC: Dump program and question configuration from production or staging PostgreSQL database to a local archive file using pg_dump.
+
+set -e
+
+export AWS_DEFAULT_REGION=us-west-2
+TIMESTAMP=$(date +%s)
+ENVIRONMENT=staging
```

Comment:
the guide says to prefer quoting bare strings.
-------

Prefer quoting strings that are “words” (as opposed to command options or path names).


\# "prefer quoting words", not compulsory
readonly USE_INTEGER='true'



https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names

Reply:
done